org/wildfly/mod_cluster/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.mod_cluster
```

Comment:
currently none, I've just documented the state created as of https://github.com/wildfly/wildfly/pull/9848